Assets/Dev/B/Script/CardSystem.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class CardSystem : MonoBehaviour
+{
+    public int amountStart;
+    public int maxAmount;
+    public Card[] drawableCards;
+    public Card[] uniqueCards;
+    public GameObject template;
+    public int x_start;
+    public int y_start;
+    public int gap;
+
+    private List<GameObject> places = new List<GameObject>();
+    private List<Card> handcards = new List<Card>();
+    private int lastIndex;
+
+    private void Awake()
+    {
+        GameObject empty = new GameObject();
+
+        for (int i = 0; i < maxAmount; i++)
+        {
+            var obj = Instantiate(empty, drawableCards[0].template.transform);
+            obj.transform.SetParent(this.transform);
+            obj.transform.position = new Vector3(x_start + ((drawableCards[0].template.GetComponentInChildren<RectTransform>().rect.width + 19 + gap) * i), y_start, obj.transform.position.z);
+            places.Add(obj);
+        }
+
+        if (amountStart <= maxAmount)
```

Comment:
These variables need to tell use what is being measured, maybe something like startingCardCount and maxCardCount